docs/source/installing.rst

```diff
@@ -73,6 +73,8 @@ Installing using PlatformIO
 
 PlatformIO is a professional collaborative platform for embedded development. It has out-of-the-box support for ESP32 SoCs and allows working with Arduino ESP32 as well as ESP-IDF from Espressif without changing your development environment. PlatformIO includes lots of instruments for the most common development tasks such as debugging, unit testing, and static code analysis.
 
+.. warning:: PlatformIO is not officially supported by Espressif. Please report any issues with PlatformIO directly to its developers `the official PlatformIO repositories <https://github.com/platformio>`_.
```

Comment:
Please report any issues with PlatformIO directly to its developers on the official `PlatformIO repositories <https://github.com/platformio>`_.

Reply:
Oops, missed that. Thanks!